app/scripts/controllers/edit/autoscaler.js

```diff
@@ -78,6 +78,21 @@ angular.module('openshiftConsole')
     var horizontalPodAutoscalerVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
     var limitRangesVersion = APIService.getPreferredVersion('limitranges');
 
+    var getHPAWarnings = function(resource, limitRanges, project, context) {
+      var scaleTarget = resource.spec.scaleTargetRef;
+      var scaleTargetVersion = APIService.objectToResourceGroupVersion(scaleTarget);
+      // if the DC still has extensions in the verison, it will fail, so we need to
+      // get the preferred version.
+      var scaleTargetPreferredVersion = APIService.getPreferredVersion(scaleTargetVersion.resource);
+      return DataService.get(scaleTargetPreferredVersion, scaleTarget.name, context).then(function(target) {
```

Comment:
I don't think we can always get the scale target because it might be a scale subresource that doesn't support GET?

Reply:
Is there a good example of that?   I'm thinking about things we have like `deploymentconfig/instantiate` or `pods/log` or `builds/clone` and can't think of a why an autoscaler would be attached.